### PR TITLE
Ppf dump

### PR DESCRIPTION
**Pull request by reginafcompton**
Handles issue #393

### CODE REVIEW

File: lots_admin/models.py [hunk 1]
```diff
@@ -142,13 +142,19 @@ def address(self):
...
 
             else:
-                return self.entity.contact_address.street
+                address_obj = self.entity.contact_address
 
         else:
-            return self.entity.address.street
+            address_obj = self.entity.address
+
+        return '{street}, {city}, {state} {zip_code}'.format(street=address_obj.street,
```
Comment (hancush) [inline lots_admin/models.py]: so smart to do this here!

File: lots_admin/views.py [hunk 2]
```diff
@@ -360,41 +360,33 @@ def org_address(ppf):
...
-        'License plate state',
-        'License plate number',
+        'Department',
+        'Requester',
+        'Date Received',
+        'Principal\'s Name',
+        'Principal\'s Address',
+        'Entity',
+        'Address',
+        'Driver\'s License Number',
```
Comment (hancush) [inline lots_admin/views.py]: i think we should retain the state for driver's license & license plate numbers.

File: lots_admin/views.py [hunk 3]
```diff
@@ -360,41 +360,33 @@ def org_address(ppf):
...
+        'Last 4 of SSN',
     ]
 
     rows = []
 
     for profile in profiles:
         organization, organization_address = org_address(profile)
         rows.append([
-            profile.application.id,
+            '',
```
Comment (hancush) [inline lots_admin/views.py]: should this be dpd?

Comment (reginafcompton) [reply lots_admin/views.py]: Kim did not indicate that in her email, but that would make sense.

**Comment by reginafcompton** [comment]
updates made!

**Pull request merged**